infra/gcp/terraform/modules/workload-identity-service-account/outputs.tf

```
@@ -0,0 +1,25 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+output "service_account" {
+  description = "The serviceaccount that was created"
+  value       = google_service_account.serviceaccount
+}
+
+output "iam_policy" {
+  description = "The serviceaccount iam_policy"
+  value       = google_service_account_iam_policy.serviceaccount_iam
```

If you want to get the policy created : 

```suggestion
  value       = data.google_iam_policy.workload_identity.policy_data

---

Again, I thought we might want the actual iam resource here, not just its policy data.  I'll rename the resource to signify the full iam resource, or bind to the policy data.  My concern with the latter is if we find a case where we need to add the policy bindings vs. set them